### ISSUE TITLE
Scrolling on WP8+

### ISSUE THREAD

**Issue by JimBobSquarePants** [t4t5/sweetalert#9]
Scrolling the page on WP8+ when an alert is present causes the overlay to scroll with a jank.

Adding the following CSS 

```
body {
    touch-action: none;
    height: 100%;
    overflow: hidden;
}
```

Will prevent this. It might be prudent though to apply it only to IE using something like the code below as it will cause the layout to jump on desktop browsers.

```
if( window.MSPointerEvent) {
   // add classes
}
```

**Comment by masteradhoc** [comment]
Hi there ( and @t4t5 soory for the late test!)
Just tested this issue. If im on the sweetalert homepage and open a regular alert, no scrolling able. If i open an sweet alert im able to scroll and even press another button while an alert is open.

So the Problem with your code:
body {
    touch-action: none;
    height: 100%;
    overflow: hidden;
}

is, that im now not even able to zoom in the page at all. Thats not that great..
See : https://jsfiddle.net/hyoLyjwc/1/ and https://jsfiddle.net/hyoLyjwc/1/embedded/result/ at WP8

**Comment by t4t5** [comment]
Well, Windows Phone 8 is not officially obsolete, but I suspect some things might have been fixed in the new 2.0 release so I'm closing for now.
